qt/place_page_dialog_user.cpp

```diff
@@ -0,0 +1,291 @@
+      value->setOpenExternalLinks(true);
+      value->setTextInteractionFlags(Qt::TextBrowserInteraction);
+
+      data->addWidget(value, row++, 1);
+    }
+
+    if (auto instagram = info.GetMetadata(feature::Metadata::EType::FMD_CONTACT_INSTAGRAM); !instagram.empty()){
+      data->addWidget(new QLabel("Instagram"), row, 0);
+
+      QLabel * value = new QLabel(QString::fromStdString("<a href='" + osm::socialContactToURL(feature::Metadata::EType::FMD_CONTACT_INSTAGRAM, std::string(instagram)) + "'>" + std::string(instagram) + "</a>"));
+      value->setOpenExternalLinks(true);
+      value->setTextInteractionFlags(Qt::TextBrowserInteraction);
+
+      data->addWidget(value, row++, 1);
+    }
+
+    if (auto twitter = info.GetMetadata(feature::Metadata::EType::FMD_CONTACT_TWITTER); !twitter.empty()){
+      data->addWidget(new QLabel("Twitter"), row, 0);
+
+      QLabel * value = new QLabel(QString::fromStdString("<a href='" + osm::socialContactToURL(feature::Metadata::EType::FMD_CONTACT_TWITTER, std::string(twitter)) + "'>" + std::string(twitter) + "</a>"));
+      value->setOpenExternalLinks(true);
+      value->setTextInteractionFlags(Qt::TextBrowserInteraction);
+
+      data->addWidget(value, row++, 1);
+    }
+
+    if (auto vk = info.GetMetadata(feature::Metadata::EType::FMD_CONTACT_VK); !vk.empty()){
+      data->addWidget(new QLabel("VK"), row, 0);
+
+      QLabel * value = new QLabel(QString::fromStdString("<a href='" + osm::socialContactToURL(feature::Metadata::EType::FMD_CONTACT_VK, std::string(vk)) + "'>" + std::string(vk) + "</a>"));
+      value->setOpenExternalLinks(true);
+      value->setTextInteractionFlags(Qt::TextBrowserInteraction);
+
+      data->addWidget(value, row++, 1);
+    }
+
+    if (auto line = info.GetMetadata(feature::Metadata::EType::FMD_CONTACT_LINE); !line.empty()){
+      data->addWidget(new QLabel("Line"), row, 0);
+
+      QLabel * value = new QLabel(QString::fromStdString("<a href='" + osm::socialContactToURL(feature::Metadata::EType::FMD_CONTACT_LINE, std::string(line)) + "'>" + std::string(line) + "</a>"));
+      value->setOpenExternalLinks(true);
+      value->setTextInteractionFlags(Qt::TextBrowserInteraction);
+
+      data->addWidget(value, row++, 1);
+    }
+
+    if (auto wikimedia_commons = info.GetMetadata(feature::Metadata::EType::FMD_WIKIMEDIA_COMMONS); !wikimedia_commons.empty()){
+      QLabel * value = new QLabel(QString::fromStdString("<a href='" + feature::Metadata::ToWikimediaCommonsURL(std::string(wikimedia_commons)) + "'>Wikimedia Commons</a>"));
+      value->setOpenExternalLinks(true);
+      value->setTextInteractionFlags(Qt::TextBrowserInteraction);
+
+      data->addWidget(value, row++, 0);
+    }
+
+    // Level fragment
+    if (auto level = info.GetMetadata(feature::Metadata::EType::FMD_LEVEL); !level.empty()){
+      addEntry("Level", std::string(level));
+    }
```

Comment:
One liners without bracers here and in other places.
```suggestion
    if (auto level = info.GetMetadata(feature::Metadata::EType::FMD_LEVEL); !level.empty())
      addEntry("Level", std::string(level));
```

Reply:
Btw, did you try to use git-clang-format? Make a commit before doing it.

Reply:
Braces are done. I will check the tool